README.md

```diff
@@ -96,30 +98,41 @@ yarn ui:build
 yarn ui:test
 ```
 
-## Configuring the environment
+# Sitemap
+
+The sitemap can be found in <./public/sitemap.xml>
+
+To update its content:
 
-The app has some default configuration, but it's highly encouraged to define your own.
+1. Edit the list of pages in <./src/sitemap.js>
+2. Run `yarn sitemap`
+3. Commit the changes to git
 
-### Local configuration
+# Configuration
+
+The app has some default configuration (in `.env` and `.env.prod` file), but it's highly encouraged to define your own `.env.local`.
 
 Make a copy of `.env` named `.env.local`, this will allow you to set your own configuration only in your local environment.
 
-#### What is the mandatory minimum to be able to run the application?
+## Supported networks (⚠️ Required)
 
-Your environment MUST have the following variables defined:
-`REACT_APP_INFURA_KEY`=YOUR_INFURA_KEY
-`REACT_APP_NETWORK_URL_1`=https://mainnet.infura.io/v3/{YOUR_INFURA_KEY}
-`REACT_APP_NETWORK_URL_5`=https://goerli.infura.io/v3/{YOUR_INFURA_KEY}
+You should set your own RPC endpoints by defining the following environment variables:
 
-Additionally, to successfully run E2E / Integration tests locally you must have;
-`INTEGRATION_TESTS_INFURA_KEY`=YOUR_INFURA_KEY
-`INTEGRATION_TESTS_PRIVATE_KEY`=YOUR_TEST_WALLET_PRIVATE_KEY
+```ini
+# Define your own RPC endpoints
+REACT_APP_NETWORK_URL_1=https://mainnet.infura.io/v3/{YOUR_INFURA_KEY}
+REACT_APP_NETWORK_URL_5=https://goerli.infura.io/v3/{YOUR_INFURA_KEY}
+REACT_APP_NETWORK_URL_100=https://rpc.gnosischain.com
```

Comment:
Since you mention the INFURA key down below, I think these examples could be generic. What do you think?

```suggestion
REACT_APP_NETWORK_URL_1=https://...  
REACT_APP_NETWORK_URL_5=https://...
REACT_APP_NETWORK_URL_100=https://...
```

Reply:
I will rebase and merge into this other PR changes, so i do all at once

Reply:
Fixed